src/deluge/io/midi/midi_follow.cpp

```diff
@@ -380,7 +380,9 @@ void MidiFollow::noteMessageReceived(MIDIDevice* fromDevice, bool on, int32_t ch
 					clipForLastNoteReceived[note] = clip;
 				}
 				else {
-					clipForLastNoteReceived[note] = nullptr;
+					if (note > 0 && note <= 127) {
+						clipForLastNoteReceived[note] = nullptr;
```

Comment:
Change to `if (note >= 0 && note <= kMaxMIDIValue)`

(Just checked and note value of zero does exist)

Reply:
Oh sorry it's not complete - I need to handle the all notes off message properly too